src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -55,6 +56,12 @@
  */
 public class BitbucketBuildStatusNotifications {
 
+    public final static String DESCRIPTION_SUCCESS = Messages.BitbucketBuildStatusNotifications_DescriptionSuccess();
```

Comment:
I think you should use inline because the language depends from the browser request. In this manner you are Forcing the language for all at class initialisation

Reply:
Thanks, changed